scripts/tdex-run-notls

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -e
+
+echo ""
+echo "starting tdexd"
+
+TDEX_BASE_ASSET="5ac9f65c0efcc4775e0baec4ec03abdde22473cd3cf33c0419ca290e0751b225"
+
+while getopts a: flag
+do
+  case "${flag}" in
+    a) TDEX_BASE_ASSET=${OPTARG};;
+    *) echo "usage: $0 [-a TDEX_BASE_ASSET]" >&2
+       exit 1 ;;
+  esac
+done
+
+echo "TDEX_BASE_ASSET: $TDEX_BASE_ASSET";
+echo ""
+
+mkdir -p tdexd
+
+docker run -it -d --name tdexd \
+  -p 9945:9945 -p 9000:9000 \
+  -v "$(pwd)/tdexd:/home/user/.tdex-daemon" \
+  -e TDEX_NO_MACAROONS=false \
+  -e TDEX_CONNECT_PROTO=http \
```

Comment:
```suggestion
  -e TDEX_CONNECT_PROTO=http \
  -e TDEX_CONNECT_ADDR=localhost \
```

Reply:
no, this would make the generated connect url to be `tdexdconnect://localhost?proot=http` which is not correct at all. It's ok to not define a connect addr, `localhost:<operator_port>` is used by default.